tests/tracing/test_fluent.py

```diff
@@ -36,10 +34,21 @@
 from mlflow.tracing.export.inference_table import pop_trace
 from mlflow.tracing.fluent import start_span_no_context
 from mlflow.tracing.provider import _get_trace_exporter, _get_tracer
+from mlflow.tracking.fluent import _get_experiment_id
 from mlflow.utils.file_utils import local_file_uri_to_path
 from mlflow.utils.os import is_windows
+from mlflow.version import IS_FULL_MLFLOW_INSTALLED
 
-from tests.tracing.helper import create_test_trace_info, get_traces
+from tests.tracing.helper import (
+    create_test_trace_info,
+    get_traces,
+    purge_traces,
+    skip_when_testing_trace_sdk,
+)
+
+# NB: mlflow.search_traces() depends on Pandas by default.
```

Comment:
Q: Should we change the default behavior of `search_traces` based on `IS_FULL_MLFLOW_INSTALLED` instead of changing the test bahevior?